packages/node_modules/@ciscospark/widget-space/src/container.js

```diff
@@ -65,6 +51,40 @@ export class SpaceWidget extends Component {
   constructor(props) {
     super(props);
     // Store activity types used in widget
+    const DEFAULT_ACTIVITY_TYPES = [
```

Comment:
I moved the activity types into the class so we could have access to `this` in order to pass props to an activity.